# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/overlay-tools/hotfix-xattr.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- overlayfs-tools-291c7f4/logic.c.vanilla	2022-01-30 19:06:38.717412282 +0100
+++ overlayfs-tools-291c7f4/logic.c	2022-01-30 19:07:04.522411253 +0100
@@ -7,7 +7,7 @@
 #include <string.h>
 #include <errno.h>
 #include <unistd.h>
-#include <attr/xattr.h>
+#include <sys/xattr.h>
 #include <attr/attributes.h>
 #include <fts.h>
 #include <libgen.h>
--- overlayfs-tools-291c7f4/main.c.vanilla	2022-01-30 19:06:38.715412282 +0100
+++ overlayfs-tools-291c7f4/main.c	2022-01-30 19:06:43.021412110 +0100
@@ -13,7 +13,7 @@
 #include <linux/limits.h>
 #include <stdbool.h>
 #include <sys/stat.h>
-#include <attr/xattr.h>
+#include <sys/xattr.h>
 #ifndef _SYS_STAT_H
   #include <linux/stat.h>
 #endif
